compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPU2DScalableTo1DScalable.cpp

```diff
@@ -87,6 +88,9 @@ class LLVMCPU2DScalableTo1DScalablePass
 };
 
 static bool opKnownToSupport2DScalableVectorizationWithArmSME(Operation *op) {
+  if (auto genericOp = dyn_cast<linalg::GenericOp>(op))
+    return LinalgOpInfo(genericOp).isTranspose() &&
+           isLinalgGeneric2DTranspose(genericOp);
```

Comment:
`LinalgOpInfo` is redundant right? 
```suggestion
    return isLinalgGeneric2DTranspose(genericOp);
```